app/src/main/java/ar/com/wolox/android/training/ui/training/login/ILoginView.java

```diff
@@ -0,0 +1,16 @@
+package ar.com.wolox.android.training.ui.training.login;
+
+/**
+ * ILoginView
+ */
+public interface ILoginView {
+
+    void onEmptyEmailError();
```

Comment:
I like it, but since the presenter will say what to do to the view, it shouldn't be named like events notifications, it should be like "orders". `showEmptyEmailError`, `showInvalidEmailError` and so.